<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日期和时间数据</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
   <property name="spacing">
    <number>1</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日期和时间数据</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,2,2,2,2,2,2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_SetCurDataAndTime">
          <property name="text">
           <string>读取当前日期和时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>字符串显示</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,2,2">
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_time"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_qDebugTime">
          <property name="text">
           <string>qDebug--Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_SetTimeFromString">
          <property name="text">
           <string>设置时间fromString</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,2,2">
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>日期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit__date"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>qDebug--date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_SetDateFromString">
          <property name="text">
           <string>设置日期fromString</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,2,2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>日期时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_dataAndtime"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
        <property name="spacing">
         <number>13</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_qDebugDateTime">
          <property name="text">
           <string>qDebug--DateTime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_SetDateAndTimeFromString">
          <property name="text">
           <string>日期时间fromString</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>日历选择</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>选择的日期：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_calendar"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
